src/persistor.ts

```diff
@@ -0,0 +1,9 @@
+export interface ReposSetupData<V>{
```

Comment:
Your persistor shouldn't know anything about your repositories. So, change this name.